src/js/components/main/grid.js

```diff
@@ -8,32 +8,42 @@ class GridView {
   #viewStore;
   #subscribeStore;
   page = 0;
-  FIRST_PAGE = 0;
-  LAST_PAGE = 3;
-  GRID_COUNT = 24;
-  constructor(gridData) {
+  viewContainer;
+  constructor(gridData, { GRID_INFO }) {
     this.#gridData = gridData;
     this.#viewStore = ViewStore;
     this.#subscribeStore = SubscribeStore;
+    this.FIRST_PAGE = GRID_INFO.FIRST_PAGE;
+    this.LAST_PAGE = GRID_INFO.LAST_PAGE;
+    this.GRID_COUNT = GRID_INFO.GRID_COUNT;
+  }
+
+  setTemplate() {
     this.viewContainer = createEl('div', 'view-container');
+    this.viewContainer.innerHTML = this.renderAllGridView();
     this.moveToPage();
+    this.onMouseGridCell();
   }
 
   render() {
```

Comment:
init() {
   this.setTemplate();
   this.store.subscribe();'
}

getViewConatiner() {
   return this.viewContainer;
}

const gridContainer = gridObj.init().getViewContainer();